data/geolocation-cn

```diff
@@ -147,6 +147,9 @@ include:category-media-cn
 # Organizations
 include:category-hospital-cn
 
+# Network Security
+include:category-network-security
```

Comment:
It is not guaranteed that only Chinese sites will be included in this category.
A sloppy solution is to rename it to `category-network-security`, hoping the renaming would not cause any other problems.

Reply:
What is the name you want to change?